cell_classification/augmentation_pipeline.py

```diff
@@ -0,0 +1,80 @@
+import imgaug as ia
+import imgaug.augmenters as iaa
+from imgaug.augmentables.segmaps import SegmentationMapsOnImage
+from imgaug.augmentables.batches import Batch
+import numpy as np
+
+
+def augment_images(images, masks, augmentation_pipeline):
```

Comment:
We'll need to make sure the tf.record follows this same format of separate stacks for images and masks

Reply:
Yep, this goes into the `prepare_example` function. We should do it anyways, because the actual mibi image will be the only thing that is stored as float, for the rest we'll use the minimal data type (i.e. uint8 for binary mask, uint16 instance_mask) to reduce i/o as much as possible.